src/plugins/presentation_util/kibana.json

```diff
@@ -18,7 +18,7 @@
     "embeddable",
     "expressions",
     "dataViews",
-    "sharedUx"
+    "sharedUX"
```

Comment:
For plugin ids we have historically kept to the most simple form of camelcase, and only capitalized the first letter in an abbreviation. Said another way, the capital just replaces the `_` in the snake_case version of the word, so `shared_ux` would be come `sharedUx`

Reply:
Oh, I was confused about which PR this is, maybe @clintandrewhall is open to a rename PR outside of this work or maybe when the number of inflight PRs is a bit lower.